connectivity/pxp_reporter_makefile/build/Makefile

```diff
@@ -0,0 +1,397 @@
+ifneq ($(strip $(ASM_DEPS)),)
+-include $(ASM_DEPS)
+endif
+ifneq ($(strip $(S_UPPER_DEPS)),)
+-include $(S_UPPER_DEPS)
+endif
+ifneq ($(strip $(C_DEPS)),)
+-include $(C_DEPS)
+endif
+endif
+
+USER_OBJS :=
+LIBS := -lble_stack_da1469x 
+
+# Templates for cleaning 
+define clean_extensions_template
+	@find . -wholename */$(strip $(1))/*.elf -type f -delete
+	@find . -wholename */$(strip $(1))/*.ld -type f -delete
+	@find . -wholename */$(strip $(1))/*.map -type f -delete
+	@find . -wholename */$(strip $(1))/*.img -type f -delete
+	@find . -wholename */$(strip $(1))/*.bin -type f -delete
+	@find . -wholename */$(strip $(1))/*.prev -type f -delete
+endef
+
+define clean_template
+
+	@echo 'Cleaning $(strip $(1))'
+	$(call clean_extensions_template, $(1))
+	-@$(RM) \
+	$(addprefix $(strip $(1)),$(OBJS)) \
+	$(SECONDARY_FLASH) \
+	$(SECONDARY_SIZE) \
+	$(addprefix $(strip $(1)),$(ASM_DEPS)) \
+	$(addprefix $(strip $(1)),$(S_UPPER_DEPS)) \
+	$(addprefix $(strip $(1)),$(C_DEPS)) 
+	@echo 'Finished cleaning $(strip $(1))'
+	@echo ' '
+
+endef
+
+################################################################################
+# Rules
+################################################################################
+
+# Allow extra symbols to be defined before compilation. Optional file.
+-include ../makefile.defs
+
+OPTIONAL_TOOL_DEPS := \
+$(wildcard ../makefile.defs) \
+$(wildcard ../makefile.init) \
+$(wildcard ../makefile.targets) \
+
+BUILD_ARTIFACT_NAME ?= pxp_reporter_makefile
+BUILD_ARTIFACT_EXTENSION := elf
+BUILD_ARTIFACT_PREFIX :=
+BUILD_ARTIFACT := $(BUILD_ARTIFACT_PREFIX)$(BUILD_ARTIFACT_NAME)$(if $(BUILD_ARTIFACT_EXTENSION),.$(BUILD_ARTIFACT_EXTENSION),)
+
+SECONDARY_FLASH += \
+$(BUILD_ARTIFACT_NAME).bin \
+
+SECONDARY_SIZE += \
+$(BUILD_ARTIFACT_NAME).siz \
+
+# All Target
+ifeq (ram, $(word 2,$(subst _, ,$(BUILD_CONFIG))))
+all: pre-build main-build
+else
+# For qspi builds, additional post-build is added to create a .img with the product and image headers
+all: pre-build main-build post-build
+endif
+
+# Main-build Target
+main-build: check-build-config $(BUILD_ARTIFACT) secondary-outputs 
+ 
+# Tool invocations
+
+# Generate .elf 
+$(BUILD_ARTIFACT): $(OBJS) $(USER_OBJS) $(OPTIONAL_TOOL_DEPS)
+	@echo 'Building target: $@'
+	@echo 'Invoking: Cross ARM C Linker'
+	@$(CC) $(LDFLAGS) \
+	 -Wl,-Map,"$(BUILD_OUTPUT_DIR)/$(BUILD_ARTIFACT_NAME).map" \
+	--specs=nano.specs --specs=nosys.specs -o "$(BUILD_OUTPUT_DIR)/$@" $(OBJS) $(USER_OBJS) $(LIBS)
+	@echo 'Finished building target: $@'
+	@echo ' '
+
+# Generate .bin 
+$(SECONDARY_FLASH): $(BUILD_ARTIFACT) $(OPTIONAL_TOOL_DEPS)
+	@echo 'Invoking: Cross ARM GNU Create Flash Image'
+	arm-none-eabi-objcopy -O binary "$(BUILD_OUTPUT_DIR)/$<"  "$(BUILD_OUTPUT_DIR)/$@" 
+	@echo 'Finished building: $@'
+	@echo ' '
+
+# Print size
+$(SECONDARY_SIZE): $(BUILD_ARTIFACT) $(OPTIONAL_TOOL_DEPS)
+	@echo 'Invoking: Cross ARM GNU Print Size'
+	@arm-none-eabi-size --format=berkeley "$(BUILD_OUTPUT_DIR)/$<"
```

Comment:
Update to:
	@$(SIZE) --format=berkeley "$(BUILD_OUTPUT_DIR)/$<"

Reply:
Update made in [1ba3fc9](https://github.com/dialog-semiconductor/BLE_SDK10_examples/pull/30/commits/1ba3fc9295fb8640ee6ef1eb723ecbc2f42c2ad1)